docs/user-guide.md

```diff
@@ -712,6 +715,28 @@ the root of the repository.
 *Note:* Currently only Strings type values are supported as parameters to
 CloudFormation templates when deploying via AWS CodePipeline.
 
+**CloudFormation Parameters in a Multi-Organization ADF Setup** 
```

Comment:
Could you change this to a sub section, like so?
That way we can reference it in GitHub issues and other sections later on.

```suggestion
#### CloudFormation Parameters in a Multi-Organization ADF Setup

```